pdart/pds4/HstFilename.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, filename: str) -> None:
         ), "Filename must be at least six characters long"
         basename2 = basename(filename)
         assert (
-            basename2[0].lower() in "ijnu"
+            basename2[0].lower() in "ijlnouwxyz"
         ), f"First char of filename {basename2!r} must be i, j, or u."
```

Comment:
This message is now wrong; please rewrite it.  Perhaps `f"First char of filename {basename2!r} must be in {ACCEPTED_INSTRUMENTS!r}"`?